src/mmw/apps/modeling/models.py

```diff
@@ -70,20 +74,37 @@ class Project(models.Model):
         null=True,
         max_length=255,
         help_text='Well-Known Area of Interest ID for faster geoprocessing')
-    uses_custom_weather = models.BooleanField(
-        default=False,
-        help_text='Whether or not this project currently uses a custom weather'
-                  ' dataset. If true, requires a non-NULL value for'
-                  ' custom_weather_dataset.')
-    custom_weather_dataset = FileField(
-        null=True,
-        upload_to=project_filename,
-        help_text='Reference path of the uploaded file.')
 
     def __unicode__(self):
         return self.name
 
 
+class WeatherType:
```

Comment:
In my first attempt I made this an `Enum`, using ideas from https://hackernoon.com/using-enum-as-model-field-choice-in-django-92d8b97aaa63. However, that led to the database having values like `WeatherType.DEFAULT`, which looked odd. Also, I did not want to casually introduce a new pattern when we had existing examples like:

https://github.com/WikiWatershed/model-my-watershed/blob/a2fe4d6e2fc035ed22cfbc5a55e91cc440248592/src/mmw/apps/modeling/models.py#L17-L23

However, I did want to group them as `WeatherType`s. This felt like the right compromise.

Reply:
This class-as-enum pattern has been used before:

https://github.com/WikiWatershed/model-my-watershed/blob/7207d544a5412b2b673297d9e8e65f0797c4c08f/src/mmw/apps/core/models.py#L69-L72